packages/dds/tree/src/simple-tree/arrayNode.ts

```diff
@@ -664,42 +672,54 @@ abstract class CustomArrayNodeBase<const T extends ImplicitAllowedTypes>
 		fieldEditor.remove(removeStart, removeEnd - removeStart);
 	}
 	public moveToStart(sourceIndex: number, source?: TreeArrayNode): void {
-		if (source !== undefined) {
-			getSequenceField(this).moveToStart(sourceIndex, getSequenceField(source));
-		} else {
-			getSequenceField(this).moveToStart(sourceIndex);
+		const isEmptyMove = sourceIndex === 0 && (source?.length === 0 || this.length === 0);
+		if (!isEmptyMove) {
```

Comment:
Currently in the empty move case (moving from empty sequence), I do not actually call the editor, since passing in index 0, 0 in an empty sequence will error when trying to generate changeset id's on a node count of 0.

Reply:
Has been addressed in this [PR](https://github.com/microsoft/FluidFramework/commit/32a6d8b7e34fd7d0b7a357c1b7f5dc72d2a2ad92)